src/Raven.Studio/typescript/common/queryUtil.ts

```diff
@@ -32,6 +32,16 @@ class queryUtil {
         }
         return "'" + name + "'";
     }
+    
+    static escapeIndexName(indexName: string): string {
+        indexName = indexName.replace(/"/g, '\\"');
+        
+        if (indexName.toLocaleLowerCase().startsWith(queryUtil.AutoPrefix) && indexName.includes("'")) {
+            return `"${indexName}"`;
+        }
+
+        return `'${indexName}'`;
```

Comment:
what if index isn't auto and contains `'``? Do we escape that propertly? Please remember this is generic function now - so it should work for **any** index name.

Reply:
as discussed, a static index cannot be created with any special characters other than: `_ - . /`